api/_bridges/across/strategy.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  BridgeStrategy,
+  GetExactInputBridgeQuoteParams,
+  BridgeCapabilities,
+  GetOutputBridgeQuoteParams,
+} from "../types";
+import { CrossSwap, CrossSwapQuotes } from "../../_dexes/types";
+import {
+  getBridgeQuoteForExactInput,
+  getBridgeQuoteForOutput,
+} from "../../_utils";
+import { buildCrossSwapTxForAllowanceHolder } from "../../swap/approval/_utils";
+import {
+  getBridgeQuoteRecipient,
+  getBridgeQuoteMessage,
+} from "../../_dexes/utils";
+import { AppFee } from "../../_dexes/utils";
+
+const name = "across";
+const capabilities: BridgeCapabilities = {
+  ecosystems: ["evm", "svm"],
+  supports: {
+    A2A: true,
+    A2B: true,
+    B2A: true,
+    B2B: true,
+    B2BI: true,
+    crossChainMessage: true,
+  },
+};
+
+export function getAcrossBridgeStrategy(): BridgeStrategy {
+  return {
+    name,
+    capabilities,
+
+    getBridgeQuoteRecipient: (crossSwap: CrossSwap) => {
+      return getBridgeQuoteRecipient(crossSwap);
+    },
+
+    getBridgeQuoteMessage: (crossSwap: CrossSwap, appFee?: AppFee) => {
+      return getBridgeQuoteMessage(crossSwap, appFee);
+    },
+
+    getQuoteExactInput: async ({
```

Comment:
Nit to match naming convention with `getQuoteForOutput`.

```suggestion
    getQuoteForExactInput: async ({
```

Reply:
https://github.com/across-protocol/frontend/pull/1850/commits/95663657d7ee95373cb6fa033b1a314a9b25d739